src/main/java/uk/gov/ons/census/casesvc/service/FulfilmentRequestProcessor.java

```diff
@@ -22,44 +28,100 @@
   private static final String CASE_NOT_FOUND_ERROR = "Case not found error";
   private static final String DATETIME_NOT_PRESENT = "Date time not in event error";
   private static final String FULFILMENT_REQUEST_RECEIVED = "Fulfilment Request Received";
+  private static final String FULFILMENT_CODE_NOT_FOUND = "Fulfilment Code not found";
+
+  private static final Set<String> individualResponseRequestCodes =
+      new HashSet<>(Arrays.asList("UACIT1", "UACIT2", "UACIT2W", "UACIT4"));
```

Comment:
These could be fully explained in Constants and then inserted here.  Also have constants in the tests.  eg.   private static final String HOUSEHOLD_INDIVIDUAL_RESPONSE_REQUEST_ENGLAND  = "UACIT1".  I know it's a mouth full, but it also makes it clear in 6 months time when you're wondering what it is.